compiler/luci/pass/src/helpers/LayerInfoMap.cpp

```diff
@@ -34,6 +34,9 @@ LayerInfoMap layer_info_map(loco::Graph *g, std::vector<LayerInfo> &layers_info)
     for (auto node : loco::active_nodes(loco::output_nodes(g)))
     {
       auto cnode = loco::must_cast<luci::CircleNode *>(node);
+      if (cnode->opcode() == luci::CircleOpcode::CIRCLEOUTPUT)
```

Comment:
To reviewers:

Output layer has the same name with the last layer, so the exception for duplicate layer name is thrown if `layers_info` has the last layer. This PR resolves the problem.

Reply:
> so the exception for duplicate layer name is thrown

Not sure but, maybe check with multiple output nodes like `CircleSplit`

Reply:
For now, Ops with multiple outputs are disabled (https://github.com/Samsung/ONE/pull/8595). Supporting Ops w/ multiple outputs can be a future task.